README.md

```diff
@@ -1,11 +1,11 @@
-# Runnable Templates
+# Templates
 
-This repo is the basis for creating your own Runnable templates for Reactr, Atmo, and Suborbital Compute. TypeScript/AssemblyScript, Rust, and Swift are supported.
+This repo is the basis for creating your own Runnable templates for E2Core and Suborbital Compute. TypeScript/AssemblyScript, Rust, and Swift are supported.
```

Comment:
s/Runnable/module
s/Suborbital Compute/Suborbital Extension Engine.

You can remove the supported languages at the end of the sentence.